### ISSUE TITLE
Compile iOS WebRTC.framework with support for vp9 codec

### ISSUE THREAD

**Issue by bennyguitar** [oney/react-native-webrtc#175]
VP9 is explicitly not compiled into webrtc with iOS by default (though it is for Android). I know CPU usage is a bit higher with VP9 vs VP8 - but mobile bandwidth makes using VP9 a very exciting possibility.

It's also pretty easy to modify `build_ios_libs.sh` in webrtc to make this a possibility.

**Comment by zxcpoiu** [comment]
@bennyguitar

thanks for recommend.
yes, it would be better we sync codecs between two platforms as possible.

1. have you tried runs vp9 on iOS and is everything alright?
2. do you know is there any reason webrtc doesn't compile VP9 on iOS by default?
3. would you might provide how to specify compiling codecs in `build_ios_libs.sh`?

and you reminds me another question, I had open an issue for it. see #177

**Comment by bennyguitar** [comment]
* VP9 runs on iOS just fine. The reason it is not included by default in iOS webrtc builds is because it has a higher CPU usage to decode/encode (it is not a hardware encoder/decoder). VP9 also has majorly increased quality for bandwidth, which makes it compelling for mobile uses despite the high CPU. Mobile connections can be less than stellar, so getting more out of each packet that comes in is a good thing.

As far as `build_ios_libs.sh`, it's super simple to compile it with VP9. In fact, it's a 1 line change. In that file (on the `branch-heads/54` branch of webrtc), you will see:

```shell
# TODO(tkchin): Add ability to pass in a flag to build vp9. Odds are mobile
# clients will not want it though because of the higher CPU usage.
LIBVPX_BUILD_VP9=0
```

Basically just change the last line to this:

`LIBVPX_BUILD_VP9=1`

And then re-run the build script, and the WebRTC.Framework that is output will now have VP9 codec support.

**Comment by zxcpoiu** [comment]
thanks @bennyguitar I will try to include it on `55`

**Comment by zxcpoiu** [comment]
this is landed in latest master

**Comment by Proxy08** [comment]
Hi, I'm using Webrtc 1.67.1  on iphone 6/7  cpu goes up and my phone gets warm up, can you explain me the process  of Compiling iOS WebRTC.framework with support for vp9 codec? please @zxcpoiu
